lib/fizzy/execute.cpp

```diff
@@ -357,6 +357,20 @@ inline void convert(OperandStack& stack) noexcept
     stack.top() = static_cast<DstT>(stack.top().as<SrcT>());
 }
 
+/// Performs a bitcast from SrcT type to DstT type.
+///
+/// This is a formality (except for f32 -> i32 where zero-extension is needed)
```

Comment:
What zero-extension?

Reply:
When we push a `i32` value to the stack we actually zero-extend it to `i64` in `Value`.

Reply:
I improved the explanation.